book/src/concepts/use_cases.md

```diff
@@ -0,0 +1,53 @@
+# Use Cases
+
+The following are examples of people who are well-suited to using a Portal Network client, like Trin.
+
+## Laptop wallet user
+
+A user has a laptop that frequently is turned off. When
+they want to transact, they can turn on Trin and connect their
+wallet to it.
+
+*Benefit*: Wallet use without reliance on third party wallet APIs.
+
+## Desktop wallet user
+
+A user has a desktop that usually on, but most of the disk is used for other things.
+When they want to transact, their wallet is already connected to their portal node.
+
+*Benefit*: Wallet use without reliance on third party wallet APIs. Contributes to
+network health without using entire disk.
+
+## Protocol experimentation
+
+A researcher looking to explore the Ethereum protocol, testing out
+specific aspects and perhaps making experimental changes to the protocol.
+
+*Benefit*: Spin up a node and play around quickly and with low cost.
+
+## Single board computer hobbyist
+
+A raspberry pi 3, or similarly-sized computer with could contribute
+to network health.
+
+Currently a raspberry pi 4 can run a full node, with consensus
+and execution clients, however this is a bit tight and requires a ~2TB SSD.
+
+*Benefit*: Learn about Ethereum, get node access and provide the
+network with additional robustness.
+
+## Mobile user
+
+Trin is not currently configured to run on mobile, however this is plausibly
+a viable and interesting use case for the future. A trin node could run as a
+background task with configurable limits on disk, CPU and bandwidth use.
```

Comment:
I am not sure if `background task`'s like mentioned here are possible on mobile without running custom OS's or rooted/jailbroken OS's.

Realistically the usecase would be Trin could only be ran well the the app Trin was embedded in is running. For example Trin being embedded into a mobile app, but Trin wouldn't be able to make requests in the background due to Mobile OS restrictions put in place to save battery life.

Reply:
Yeah, this is another one I didn't really take a pass at, besides pointing out that it's for the future. I agree it could use more of a touch-up though 👍🏻